Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -768,11 +768,11 @@ void unified_bed_leveling::shift_mesh_height() {
     TERN_(EXTENSIBLE_UI, ExtUI::onMeshUpdate(best.pos, ExtUI::G29_START));
     do {
       if (do_ubl_mesh_map) display_map(param.T_map_type);
-
+      TERN_(DWIN_LCD_PROUI, dwinRedrawScreen());
```

Comment:
This shouldn't be needed because `MarlinUI::finish_status` already calls `dwinCheckStatusMessage()` for ProUI.

Reply:
In general, the displays that have code inserted throughout Marlin need to be given an `ExtUI` layer, and any situations where there is no `ExtUI` method for the event, a method should be added to `ExtUI` for that new event. At some point before the next release we'll also want to bring in the latest updates to ProUI, at which point it will be easier to go forward adding an ExtUI that both we and Miguel can integrate.